packages/create-audius-app/test/use-temp-dir.ts

```diff
@@ -0,0 +1,24 @@
+import fs from 'fs-extra'
+import os from 'os'
+import path from 'path'
+
+export async function useTempDir(
```

Comment:
So vitest likes a test directory at root instead of with the file under test?

Reply:
Not sure I completely understand the question

This file is used to create temp directories to install the examples in for each test. I've put all the test utilities in this `test` directory.

In the nextjs repo this folder does live at the root because they have a bunch of e2e tests that test multiple packages. I think we may want to move in that direction too. But vitest can handle any structure we throw at it really